src/Views/Shared/_Layout.cshtml

```diff
@@ -33,6 +33,7 @@
                     <li><a asp-area="" asp-controller="Home" asp-action="Index">Home</a></li>
                     <li><a asp-area="" asp-controller="Home" asp-action="About">About</a></li>
                     <li><a asp-area="" asp-controller="Home" asp-action="Contact">Contact</a></li>
+                    <li><a asp-area="" asp-controller="Movie" asp-action="Create">Movie</a></li>

```

Comment:
Nice job adding a button. Remember you will need to update this when the movie list view is created